vault/resource_nomad_secret_backend_test.go

```diff
@@ -0,0 +1,101 @@
+package vault
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/hashicorp/terraform-provider-vault/util"
+	"github.com/hashicorp/vault/api"
+)
+
+func TestAccNomadSecretBackend(t *testing.T) {
+	backend := acctest.RandomWithPrefix("tf-test-nomad")
+	address, token := util.GetTestNomadCreds(t)
```

Comment:
Does vault verify the connection to nomad with the address and token when set? Or could these be test values and run in CI? Something like:

```suggestion
	address := "https://test.address"
	token := "test-token"
```

Same question on the other nomad acceptance tests.

Reply:
This should be the value of an actual token because we'll be running this against a real Nomad system. Nomad ACL bootstrap tokens must be generated, we can't assign it a dev static value like we do in Vault.